run_tests.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Run tests for the ActiveTableXBlock.
+
+This script is required to run the Selenium tests inside the xblock-sdk workbench
+because the workbench SDK's settings file is not inside any python module.
```

Comment:
I'll have to look into this a bit myself.  I tried briefly to get the Selenium tests running without this script, and when that failed, I simply copied the script from our Problem Builder XBlock (which I didn't write), which just worked, so I didn't give it too much thought.